src/Mapbender/DigitizerBundle/Component/HttpHandler.php

```diff
@@ -75,8 +75,7 @@ protected function getUpdateMultipleActionResponseData(Element $element, Request
             $schema = $this->schemaFilter->getSchema($element, $schemaName);
             $feature = $schema->getRepository()->getById($featureData['idInSchema']);
             if (!$feature) {
-                // uh-oh!
-                continue;
+               $feature = $schema->getRepository()->itemFactory();
```

Comment:
is this guaranteed to always return an non-null feature?

Reply:
Yes, that's what itemFactory() is supposed to do. In the current digitizer setup, it's not possible to call this method on features that haven't been materialized in the database yet. However, I need this functionality for the file upload feature.